plugins/github-pull-requests-board/src/utils/types.tsx

```diff
@@ -31,12 +31,36 @@ export type GraphQlPullRequests<T> = {
   };
 };
 
+export type GraphQlUserPullRequests<T> = {
+  user: {
+    pullRequests: {
+      edges: T;
```

Comment:
Hm, I thought this would be `nodes` if it's a T, right? The `edges` shape is hard coded to be like `{ node: T, cursor: string }` basically, isn't it? Basically you could make a utility type `pullRequests: Connection<T>` where `Connection` follows the connection graphql spec and has nodes, edges, and pageInfo in it

Reply:
Yeah I see below that these types are abstracted oddly, the fact that `PullRequestsNumberAndOwner` has `node` at its root makes me think that you really want to invent the Connection type and then removing `node` from `PullRequestsNumberAndOwner`.

I'll also mention that there exists a public type lib for octokit that has all of this in it already.

Reply:
Just to confirm, you'd like me to adjust things to be more like:

```tsx
export type Connection<T> = {
  edges: [
    {
      node: T
    }
  ];
  pageInfo: {
    hasNextPage: boolean;
    endCursor?: string;
  };
};

export type GraphQlUserPullRequests<T> = {
  user: {
    pullRequests: Connection<T>
};

export type PullRequestsNumberAndOwner = {
  number: number;
  repository: Repository;
};
```